ansible/roles/test/tasks/fdb_mac_expire.yml

```diff
@@ -0,0 +1,109 @@
+- fail: msg="testbed_type is not defined"
+  when: testbed_type is not defined
+
+- fail: msg="testbed_type {{test_type}} is invalid"
+  when: testbed_type not in ['t0', 't0-64', 't0-64-32', 't0-116']
+
+- include_vars: "vars/topo_{{testbed_type}}.yml"
+
+- name: Expand properties into props
+  set_fact: props="{{configuration_properties['common']}}"
+
+- name: Gather minigraph facts about the device
+  minigraph_facts: host={{inventory_hostname}}
+
+- name: Copy tests to PTF
+  copy: src=roles/test/files/ptftests dest=/root
+  delegate_to: "{{ptf_host}}"
+
+- name: Copy FDB information file to PTF
+  template: src=roles/test/templates/fdb.j2 dest=/root/fdb_info.txt
+  delegate_to: "{{ ptf_host }}"
+
+- name: clear fdb all
+  shell: clear fdb all
+
+- name: copy current switch.json from docker to host
+  shell: docker cp swss:/etc/swss/config.d/switch.json .
+
+- name: set fdb value to "{{fdb_aging_time}}"
```

Comment:
Where is the fdb_aging_time varible initialized? If no value is assigned to it via CLI, then the test would be failed with undefined variable. It would be better to assign it a default value in this ansible yml when it is not defined. 
To limit the test execution time, it would be better to assign it a small value, say 60.

Reply:
I think, it is perfectly fine if test fails when required parameters are not passed. But it is perfectly fine to set it to default when it is not already defined.